### ISSUE TITLE
[Bug]: Update on Ubuntu causes program to not load, and Some Subscribed Channel icons on left side of window show broken images until clicked on

### ISSUE THREAD

**Issue by swilly97** [FreeTubeApp/FreeTube#6726]
### Guidelines

- [x] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [x] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [x] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the bug report I want to file, without success.
- [x] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [x] This issue contains only one bug.

### Describe the bug

1. Installed the latest 0.23.1
2. Freetube wouldn't open. Had to run command "sudo chmod 4755 /opt/FreeTube/chrome-sandbox" from user nukeador from issue #5952 to get Freetube to open.
3. On the left side of the screen where the subscribed channel icons appear, random channels have no image showing, just partial text of the channel name. Clicking on the channel icon causes the image to appear properly.

### Expected Behavior

The icons should load for the subscribed channels.

### Issue Labels

visual bug, accessibility issue

### FreeTube Version

v.0.23.1

### Operating System Version

Ubuntu MATE 24.04

### Installation Method

.deb

### Primary API used

Local API

### Last Known Working FreeTube Version (If Any)

v.0.22.1

### Additional Information

![Image](https://github.com/user-attachments/assets/a2e9e59b-ea72-42d6-9a7d-b8c6f75fdaef)

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Please Launch the application, open the dev tools, ctrl + shift +i and check if there are any red errors in the console

**Comment by swilly97** [comment]
![Image](https://github.com/user-attachments/assets/1be00bfa-eb5f-4d34-97c7-59170dd01ff6)

**Comment by ArthurKun21** [comment]
it also appears on mac

Build 5582

43065df3ddae3ba15c229ed2ce62ef901dffd8f8

![Image](https://github.com/user-attachments/assets/9a16d1d8-5290-4a91-ae18-20920df37493)

```js
1-zNalzhfIP2LSWrGIT4l-9P8MjUhgHCXGB1Q6kHibyNlRvafBYQMfoI5qnSk1L5V8N705J4fw=s35-c-k-c0x00ffffff-no-rj:1 
        
        
       Failed to load resource: the server responded with a status of 404 ()
U6INDzr16ANqJ-uqyB3naaAh1TQ6G0NxfYeN3sj5gR9vmR_oqmDj5JulGjJ74P7mSQfR6FBT2c8=s35-c-k-c0x00ffffff-no-rj:1 
        
        
       Failed to load resource: the server responded with a status of 404 ()
dDHOkfh5uTfelqCEGNdBOadDVs1elmguSYaFhYME2Xr_gbfBNPidmAyTv4AsiGrUxiybbpsH8w=s35-c-k-c0x00ffffff-no-rj:1 
        
        
       Failed to load resource: the server responded with a status of 404 ()
58uF1apTqtovpieDrBCSF5pUeNU2MA6n34bdj2EBEVnySKm-VvEznGNt2puk4v4tAnA3AYh5JA=s35-c-k-c0x00ffffff-no-rj:1 
        
        
       Failed to load resource: the server responded with a status of 404 ()
renderer.js:2 using static file for invidious instances
fetchInvidiousInstances @ renderer.js:2
lSwLKpd-BGsaDwVDNP_pgUcf4rg3akrE7p5FjAWzv3w4oR7bBkx22QhOc7ulHLQw6ZVTg-RE=s35-c-k-c0x00ffffff-no-rj:1 
        
        
       Failed to load resource: the server responded with a status of 404 ()
```

**Comment by absidue** [comment]
This doesn't seem like anything out of the ordinary. Each channel thumbnail has a unique URL so when a channel changes their thumbnail, the old URL becomes invalid (presumably because the image it points to has been deleted from the Google servers) and FreeTube has no way of knowing about the new thumbnail URL until another part of the app requests information about the channel from YouTube.

It's likely just a coincidence that it happened around the same time as the update or if you are the kind of person to leave FreeTube open for ages it probably had the old image cached and when you closed FreeTube to update it, it removed the outdated cached image.
